app.go

```diff
@@ -133,7 +135,7 @@ func pageForReq(app *app, r *http.Request) page.StaticPage {
 	p := page.StaticPage{
 		AppCfg:  app.cfg.App,
 		Path:    r.URL.Path,
-		Version: "v" + softwareVer,
```

Comment:
We can leave this as-is if we remove the **v** from `$(GITREV)` as mentioned before.